contracts/upgradeable-polymesh-ink/Cargo.toml

```diff
@@ -15,7 +15,8 @@ ink = { version = "4.3", default-features = false }
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
 scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
 
-polymesh-api = { version = "3.5", default-features = false, features = ["ink"] }
+polymesh-api = { version = "3.6", default-features = false, features = ["ink"] }
+polymesh-api-ink = { version = "1.2.2", default-features = false, features = ["use_call_runtime_with_error"] }
```

Comment:
We shouldn't enable this by default.  We can add a feature flag to this file to allow turning it on.  After this PR is merged and testnet/mainnet has it included, we can make it the default.